src/liballoc_system/lib.rs

```diff
@@ -19,6 +19,7 @@
             issue = "27783")]
 #![feature(allocator)]
 #![feature(staged_api)]
+#![cfg_attr(target_os = "redox", feature(libc))]
```

Comment:
Note that `ralloc` is used as the default allocator in `librust_back`. What we have done here is to ensure that `liballoc_system` builds, as it is a required dependency of `libstd`. We never link to it, however.